src/libs/actions/User.ts

```diff
@@ -61,7 +61,7 @@ Onyx.connect({
     },
 });
 
-let myPersonalDetails: OnyxEntry<OnyxPersonalDetails> | EmptyObject = {};
+let myPersonalDetails: OnyxEntry<OnyxPersonalDetails> | EmptyObject | null = {};
```

Comment:
```suggestion
let myPersonalDetails: OnyxEntry<OnyxPersonalDetails>;
```

Reply:
Same as in described in https://github.com/Expensify/App/pull/42772#discussion_r1637117945, though i improved it a bit by removing unnecessary `null` and just fallback to `{}` as per initial value